source/javascripts/controllers/_MainController.js.erb

```diff
@@ -454,6 +455,19 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 				});
 			}
 
+			function updateLastWorkflowMetadata(){
+				const appSlug = appService.appDetails.slug;
+				const dateToday = new Date().toString();
+			
+				requestService.updateCurrentUserMetadata({
+					'last_workflow_edited_date': dateToday,
+					'last_workflow_edited_app': appSlug
```

Comment:
The api accepts setting two metadata values at once? This function might allow it but I would confirm that the actual http api is behaving as expected

Reply:
It only accepts one metadata key per call. This is due to the way it's processed on the backend (there is no controller or model functionality to accept multiple metadata keys). I had created hook called `useUserAllMetadata` to do this on the frontend, but it still would have only been chained requests using `Promise.all`. It works fine with the separate calls, but if more than two metadata items are needed in the future, a rewrite of how this is handled in the backend would be a good idea.